frontend/src/container/BillingContainer/BillingContainer.tsx

```diff
@@ -336,74 +327,84 @@ export default function BillingContainer(): JSX.Element {
 		updateCreditCard,
 	]);
 
+	const BillingUsageGraphCallback = useCallback(
+		() =>
+			!isLoading ? (
+				<BillingUsageGraph data={apiResponse} billAmount={billAmount} />
+			) : (
+				<Card className="empty-graph-card" bordered={false}>
+					<Spinner size="large" tip="Loading..." height="35vh" />
+				</Card>
+			),
+		[apiResponse, billAmount, isLoading],
+	);
+
 	return (
 		<div className="billing-container">
-			<Row
-				justify="space-between"
-				align="middle"
-				gutter={[16, 16]}
-				style={{
-					margin: 0,
-				}}
+			<Flex vertical style={{ marginBottom: 16 }}>
+				<Typography.Text style={{ fontWeight: 500, fontSize: 18 }}>
+					Billing
+				</Typography.Text>
+				<Typography.Text color={Color.BG_VANILLA_400}>
+					Manage your billing information, invoices, and monitor costs.
+				</Typography.Text>
+			</Flex>
+
+			<Card
+				bordered={false}
+				style={{ minHeight: 150, marginBottom: 16 }}
+				className="page-info"
 			>
-				<Col span={20}>
-					<Typography.Title level={4} ellipsis style={{ fontWeight: '300' }}>
-						{headerText}
-					</Typography.Title>
-
-					{licensesData?.payload?.onTrial &&
-						licensesData?.payload?.trialConvertedToSubscription && (
-							<Typography.Title
-								level={5}
-								ellipsis
-								style={{ fontWeight: '300', color: '#49aa19' }}
-							>
-								We have received your card details, your billing will only start after
-								the end of your free trial period.
-							</Typography.Title>
-						)}
-				</Col>
-
-				<Col span={4} style={{ display: 'flex', justifyContent: 'flex-end' }}>
+				<Flex justify="space-between" align="center">
+					<Flex vertical>
+						<Typography.Title level={5} style={{ marginTop: 2, fontWeight: 500 }}>
+							Enterprise Cloud{' '}
```

Comment:
This also should be conditional

Reply:
Billing page wont be available for other than Enterprise